cdhweb/blog/templates/blog/blogpost_detail.html

```diff
@@ -25,24 +23,24 @@
 {% block main %}
 <article typeof="schema:BlogPosting" class="blogpost">
     <header>
-        <h1 property="schema:headline">{{ post.title }}</h1>
-        <meta property="schema:url schema:mainEntityOfPage" content="{% pageurl post %}"/>
+        <h1 property="schema:headline">{{ page.title }}</h1>
+        <meta property="schema:url schema:mainEntityOfPage" content="{% pageurl page %}"/>
```

Comment:
does pageurl return a full url? or can we use `get_full_url` here like you recommended for the site url in the footer? in my experience stuff this kind of metadata does require full urls

Reply:
it's supposed to "intelligently" decide when to return a full url, but probably better to be safe

Reply:
I think the "intelligence" is for cross-site / multisite linking; I doubt it could tell that's needed here